src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md

```diff
@@ -176,6 +176,7 @@ This release includes 1234 merged PRs by 97 authors.
 [#124681]: https://github.com/cockroachdb/cockroach/pull/124681
```

Comment:
Some links were broken in this alpha.1 and I realized they were missing.

Reply:
is this due to a bug in the `release-notes.py` script? if so would you please consider filing a DOC issue with the 'Product Area = Edu Eng - Docs Toolchain' and ping Eeshan ?

Reply:
I don't think so, I actually think I did these notes and just did a poor job of removing things!